webpack.config.js

```diff
@@ -38,10 +38,10 @@ const commonConfig = {
   },
   optimization: {
     minimizer: [
-      new UglifyJsPlugin({
-        parallel: true,
-      }),
-    ],
+      new TerserPlugin({
+      parallel: true,
+    }),
+  ],
```

Comment:
Please use the following config to avoid building with comments (`parallel` is `true` by default). Otherwise looks good.

```suggestion
      new TerserPlugin({
        extractComments: false,
        terserOptions: {
          format: {
            comments: false,
          },
        },
      }),
    ],
```